scripts/ci/make/util

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env bash
+
+#########################################################
+###   COMMON UTILITY FUNCTIONS - ALPHABETICAL ORDER   ###
+#########################################################
+
+cleanup() {
+    rm -rf "${VENV}/tmp"
+}
+
+clean_container() {
+    print_colorized ERROR "deleting ${TEST_IMAGE_NAME}"
+    sleep 3
+    docker rmi "${TEST_IMAGE_NAME}"
+}
+
+clean_python_env() {
+    print_colorized ERROR "deleting virtualenv & build data"
+    sleep 3
+    rm -rf "${VENV}" \
+        *.egg-info \
+        dist \
+        build
+    find . -name '*.pyc' -exec rm -f {} \;
+}
+
+clean_tests() {
+    print_colorized ERROR "deleting testing temp data"
+    sleep 3
+    rm -rf .tox
+    rm -f tox.log
+    rm -rf .pytest_cache
+}
+
+docker_compose_down() {
+    docker-compose -f scripts/ci/docker-compose-ci.yaml down
+}
+
+docker_compose_up() {
+    docker-compose -f scripts/ci/docker-compose-ci.yaml up -d
+}
+
+install() {
+    print_colorized WARN "installing ${GIT_REPO} to ${VENV}"
+    pip install .
+}
+
+install_dev() {
+    print_colorized WARN "installing ${GIT_REPO} in editable mode to ${VENV}"
+    pip install --editable .
+}
+
+install_cluster_deps() {
+    local arch=$(uname | tr A-Z a-z)
+
+    if [[ ! -x "${VENV}/bin/kind" ]]; then
+        print_colorized WARN "installing kind"
+        curl -qsSLo "${VENV}/bin/kind" "https://github.com/kubernetes-sigs/kind/releases/download/${KIND_VERSION}/kind-${arch}-amd64"
+        chmod +x "${VENV}/bin/kind"
+    fi
+    if [[ ! -x "${VENV}/bin/helm" ]]; then
+        print_colorized WARN "installing helm"
+        curl -sSL "https://get.helm.sh/helm-${HELM_VERSION}-${arch}-amd64.tar.gz" | tar xzf - -C "${VENV}/bin" --strip-components=1 "${arch}-amd64/helm"
+        chmod +x "${VENV}/bin/helm"
+    fi
+    if [[ ! -x "${VENV}/bin/kubectl" ]]; then
+        print_colorized WARN "installing kubectl"
+        curl -sSLo "${VENV}/bin/kubectl" "https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_VERSION}/bin/${arch}/amd64/kubectl"
+        chmod +x "${VENV}/bin/kubectl"
+    fi
+}
+
+install_testing_deps() {
+    print_colorized WARN "installing dependencies: $*"
+    mkdir -p .tox
```

Comment:
why is this directory being created?

Reply:
that's not needed anymore - forgot to remove it.